app/controllers/application_controller/performance.rb

```diff
@@ -203,6 +203,7 @@ def perf_set_or_fix_dates(options)
     edate_daily = edate.hour < 23 ? edate - 1.day : edate
     options[:edate_daily] = edate_daily
 
+    options[:typ] = "Hourly" if options[:typ] == "Daily" && edate_daily < sdate_daily
```

Comment:
In the commit there is this description
"Fix - back to hourly date range if daily only has an incomplete day"
Which IMO is good.
This fix is related to LiveMetrics model.
This aspect of performance.rb is harmless for others usecases but only applies to LiveMetrics.
I don't see a problem to have them in the same PR.
Separate them is an extra effort that I don't see comparing with other similar PRs.